integrated_channels/sap_success_factors/transmitters/courses.py

```diff
@@ -42,17 +44,34 @@ def transmit_block(self, serialized_payload):
 
         return status_code, body
 
-    def transmit(self, payload):
+    def transmit(self, course_exporter):
         """
         Send a course data import call to SAP SuccessFactors using the client.
 
         Args:
-            payload (SapCourseExporter): The OCN course exporter object to send to SAP SuccessFactors
+            course_exporter (SapCourseExporter): The OCN course exporter object to send to SAP SuccessFactors
         """
+        CatalogTransmissionAudit = apps.get_model(  # pylint: disable=invalid-name
+            app_label='sap_success_factors',
+            model_name='CatalogTransmissionAudit'
+        )
+
+        try:
```

Comment:
Can you add a comment describing the exception case, when/how often it is expected to occur, etc.